DFC.Composite.Shell/Views/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -1,18 +1,20 @@
 <environment include="Development">
-    <script src="~/lib/jquery-validation/jquery.validate.js"></script>
-    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+    <script src="~/lib/jquery-validation/jquery.validate.js" nws-csp-add-nonce="true"></script>
+    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js" nws-csp-add-nonce="true"></script>
```

Comment:
Delete these lines, should be using dfc-digital-assets or CDN

Reply:
Removed